tests/files/coredns-snippet.template

```diff
@@ -1,4 +1,4 @@
-    x.asynchrono.us:53 {
+    samba1.sink.test:53 {
```

Comment:
I changed the container repo PR https://github.com/samba-in-kubernetes/samba-container/pull/40 to use `domain1` instead `samba1`. One problem is that the dc's netbios name is `samba1` too, which is problematic.

In general, I prefer using names that tell what kind of object the named object is, e.g. `domain1`, `domain2` etc for domain names `dc1`, `dc2` etc for domain controllers. (I changed the pr to also update the dc's netbios name fwiw.)